pkgs/applications/editors/quartus-prime/default.nix

```diff
@@ -32,6 +32,7 @@ in buildFHSUserEnv rec {
     fontconfig = pkgs.fontconfig.override { inherit freetype; };
     libXft = pkgs.xorg.libXft.override { inherit freetype fontconfig; };
   in [
+    libxcrypt
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** It should come after libudev in 'common' session?

Reply:
Yes it should, i'll change that.